client/scss/elements/_forms.scss

```diff
@@ -105,6 +105,10 @@ select,
   }
 }
 
+.tagit {
+  background-color: $color-white;
+}
```

Comment:
This is the one mandatory change so we can remove all jQuery UI images – adding a background 😅 .